### PR TITLE
ISSD-1743 [Marketplace] Add Offering Types for DXP Apps

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/ProvideAppBuildPage/ProvideAppBuildPage.tsx [hunk 1]
```diff
@@ -125,50 +125,47 @@ export function ProvideAppBuildPage({
...
-						name: managedOption.name,
-						vocabulary: 'Marketplace Liferay Platform Offering',
-					});
+		if (fullyManagedOption) {
+			fullyManagedOption.map((managedOption) => {
+				newCategories.push({
+					externalReferenceCode: managedOption?.externalReferenceCode,
+					id: managedOption.id,
+					name: managedOption.name,
+					vocabulary: 'Marketplace Liferay Platform Offering',
```
Comment (silasbsi) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/ProvideAppBuildPage/ProvideAppBuildPage.tsx:163]: Could be 'Marketplace Liferay Platform Offering' used as a constant value?

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/ProvideAppBuildPage/ProvideAppBuildPage.tsx (hunk 1 above)
Comment (claricebrasil) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/ProvideAppBuildPage/ProvideAppBuildPage.tsx:163]: Done!

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/ProvideAppBuildPage/ProvideAppBuildPage.tsx
[the remark this answers is not in the document]
Comment (claricebrasil) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/ProvideAppBuildPage/ProvideAppBuildPage.tsx:1]: Nice! Changes done, Thank you

**Comment by claricebrasil** [comment]
ci:test:sf

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @claricebrasil

**Review by silasbsi — CHANGES_REQUESTED** [review]
Good job

**Review by silasbsi — APPROVED** [review]

**Comment by claricebrasil** [comment]
ci:test:sf

**Review by juliocezar96 — APPROVED** [review]
Good job! 🤙

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

Very very good @claricebrasil

**Pull request closed without merging**